### ISSUE TITLE
Automatically sending articulation selection

### ISSUE THREAD

**Issue by StewartKUK** [jtackaberry/reaticulate#205]
It would be really great that whenever a track is selected (with reaticulate configured) the selected articulation midi is resent to the midi device. This would ensure that the instrument does not get out of sync and will also allow multiple midi tracks for the same midi instrument. The multi midi tracks is necessary for employing track delays correctly.

**Comment by jtackaberry** [comment]
Interestingly, I think for me this would be an anti-feature, because I find I'm playing a lot with articulations and different instruments hunting for an appropriate blended sound, and having things reset on me as I select tracks would be frustrating.

But also it wouldn't probably wouldn't be difficult to implement, and tuck behind an optional setting.

**Comment by StewartKUK** [comment]
Dont you select the articulations from reaticulate? There is no need to
touch the plugins directly, the only problem is that plugins can easily get
out of sink with reaticulate which is a real problem especially with VEPro
etc. It should be optional via a configuration setting anyway, in case it
really does upset unconventional use cases.

**Comment by jtackaberry** [comment]
> Dont you select the articulations from reaticulate? 

Yes, true, for some reason I had it in my head that it would effectively chase the last articulation like REAPER does on playback, not just replay the selected articulation.  Which of course makes much less sense. :)
